haskell/defs.bzl

```diff
@@ -380,6 +396,28 @@ def haskell_test(
         **kwargs
     )
 
+    repl_kwargs = {
+        attr: kwargs[attr]
+        for attr in ["tags"]
+        if attr in kwargs
+    }
+    native.alias(
```

Comment:
Same thing about backward compatibility.

Reply:
Same reasoning as in https://github.com/tweag/rules_haskell/pull/1230#discussion_r373990927